docs/EN_US/ECLLanguageReference/ECLR_mods/BltInFunc-BUILD.xml

```diff
@@ -894,34 +894,113 @@ BUILD(FilterDsLib1);
             compression on the payload. The resulting index can be smaller
             than using lz4.</entry>
           </row>
+
+          <row>
+            <entry><emphasis role="bold"><emphasis
+            role="bold">'inplace:lz4s'</emphasis> </emphasis></entry>
+
+            <entry>Causes inplace compression on the key fields and lz4s
+            compression on the payload. This uses the stream LZ4 API to avoid
+            recompressing the data and reduce the index build times.</entry>
+          </row>
+
+          <row>
+            <entry><emphasis role="bold"><emphasis
+            role="bold">'inplace:lz4shc'</emphasis> </emphasis></entry>
+
+            <entry>The default compression for inplace indexes in versions
+            after versions 9.6.90, 9.8.66, and 9.10.12. Causes inplace
+            compression on the key fields and lz4shc compression on the
+            payload. This uses the stream LZ4 API to avoid recompressing the
+            data and reduce the index build times.</entry>
+          </row>
         </tbody>
       </tgroup>
     </informaltable>
 
-    <para>The inplace index compression format (introduced in version 9.2.0)
-    improves compression of keyed fields and allows them to be searched
-    without decompression. The original index compression implementation
-    decompresses the rows when they are read from disk.</para>
+    <para>The lz4s and lz4hc inplace index compression formats (introduced in
```

Comment:
The previous text is still relevant, and should be restored.

I'm not sure it needs as much information about the version information.  The version information tied to the compression type should be associated with it.

E.g. lz4s and lz4shc compression was introduced in versions 9.6.90, 9.8.66 and 9.10.12.  If these formats are used, the platform reading must be one of those versions or later.